Examples/Tests/galilean/analysis.py

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env python
```

Comment:
Could you add a comment to explain what this test checks?
See for instance: https://github.com/ECP-WarpX/WarpX/blob/master/Examples/Tests/particles_in_PML/analysis.py

Reply:
Done in dd39e451e85.